curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65ca2a52d579b22feb89177f.md

```diff
@@ -1,13 +1,13 @@
 ---
-id: 655a59607b0d2e9b0f5d69e5
-title: Step 61
+id: 65ca2a52d579b22feb89177f
+title: Step 62
 challengeType: 20
-dashedName: step-61
+dashedName: step-62
 ---
 
```

Comment:
I would swap the last two steps. So we can test if `delete` has deleted the key `40` and then see the inorder traversal as the last step.

Reply:
From what I understood, this should be the flow?

1. Delete `40`
2. Search `40`
3. Print tree after deleting `40`

Currently it is: delete, print, search.

Reply:
Yes, this is what I meant.